horizons/util/python/decorators.py

```diff
@@ -97,12 +97,11 @@ def __call__(self, *args, **kwargs):
 					del self.cache_dates[key]
 					return self(*args, **kwargs)
 			else:
-				self.cache_dates[key] = time.time() # new entry
+				self.cache_dates[key] = time.time()  # new entry
```

Comment:
In the file are already inline comments with more then one space:
https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/util/python/decorators.py#L117
https://github.com/unknown-horizons/unknown-horizons/blob/master/horizons/util/python/decorators.py#L136